giphy_cli.py

```diff
@@ -1,31 +1,83 @@
+import os
 import click
 import requests
 
-API_KEY = "tFRkU1RvdbGFdp76FKKBuHvMRPpCrmM7"  # Use your API key
```

Comment:
Your `API_KEY` is now a permanent part of your git history. You'll need to remove it.

[Here is a guide on different approaches to doing it](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository)